pkg/operator/nfs/controller_test.go

```diff
@@ -25,11 +25,13 @@ import (
 	"github.com/rook/rook/pkg/operator/k8sutil"
 	testop "github.com/rook/rook/pkg/operator/test"
 	"github.com/stretchr/testify/assert"
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes/fake"
 )
 
+const appName = "nfs-server-X"
```

Comment:
is this intended to be a format string instead of a hard coded X?

Reply:
nope its just a random name.